doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -74,8 +75,21 @@ lose their role as parameters. For example:
 
    Notation "'IF' c1 'then' c2 'else' c3" := (c1 /\ c2 \/ ~ c1 /\ c3) (at level 200, right associativity).
 
-Symbols that start with a single quote with 3 or more characters must be single quoted.
-For example, the symbol `'ab` is represented by `''ab'` in the notation string.
+Symbols that start with a single quote and followed by at least 2
+characters must be single quoted.  For example, the symbol `'ab` is
+represented by `''ab'` in the notation string. Strings can be used in
+notations: the start and end of a string symbol in a notation string
+must be doubled double quotes. Spaces occurring in these strings are
+part of the string symbol and do not contribute to the separation
+between tokens. To refer to double quotes in these strings, use four
+double quotes (e.g. the notation :g:`"A ""I'm a """"infix"""" string symbol"" B"`
+defines an infix notation whose infix symbol is the string
+:g:`"I'm a ""infix"" string symbol"`). Symbols are allowed to contain
+strings without being strings themselves but such notations can be
```

Comment:
Can you give an example of a symbol that contains a string but it itself a string?

Reply:
It should have been "Symbols may contain double quotes", not "Symbols may contain strings", sorry.

I added an example though.